judge/migrations/0132_no_self_vote.py

```diff
@@ -0,0 +1,30 @@
+from django.db import migrations
+from django.db.models import F, OuterRef, Subquery, Sum
+from django.db.models.functions import Coalesce
+
+
+def delete_self_votes(apps, schema_editor):
+    Comment = apps.get_model('judge', 'Comment')
+    CommentVote = apps.get_model('judge', 'CommentVote')
+
+    CommentVote.objects.filter(voter=F('comment__author')).delete()
+    Comment.objects.update(
+        score=Subquery(
+            Comment.objects.filter(
+                id=OuterRef('id')
+            ).annotate(
+                foo=Coalesce(Sum('votes__score'), 0)
+            ).values('foo')[:1]
+        )
+    )
```

Comment:
Nice, but I probably wouldn't call it `foo`.

Reply:
Renamed to `newscore`